src/spikeinterface/exporters/to_ibl.py

```diff
@@ -0,0 +1,305 @@
+from __future__ import annotations
+
+import os
+import shutil
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+import numpy.typing as npt
+from scipy.signal import welch
```

Comment:
`import scipy` should be done within the functions using it because it's not a mandatory requirement (see failing tests)